src/components/scenes/ScanScene.js

```diff
@@ -45,6 +45,12 @@ export class Scan extends React.Component<Props> {
                 <T style={styles.bottomButtonText}>{s.strings.fragment_send_flash}</T>
               </View>
             </TouchableHighlight>
+            <TouchableHighlight style={styles.bottomButton} onPress={this._onToggleAddressModal} underlayColor={THEME.COLORS.SECONDARY}>
+              <View style={styles.bottomButtonTextWrap}>
+                <IonIcon style={styles.addressBookIcon} name="ios-book" size={scale(24)} />
```

Comment:
The icon should not be an address book, but instead look like the "enter" icon in the send scene, the one used in this line - https://github.com/EdgeApp/edge-react-gui/blob/b7ba44615690d09e721a40189fb517d223af376e/src/components/themed/AddressTile.js#L275